test/libsolidity/semanticTests/array/copying/copy_dynamically_sized_storage_array.sol

```diff
@@ -0,0 +1,12 @@
+contract C {
+  function() external [] s0;
+  function() external [] s1;
+  constructor() {
+    s0 = s1;
+  }
+}
+// ====
+// compileToEwasm: also
+// compileViaYul: also
+// ----
+// () -> FAILURE
```

Comment:
"FAILURE" does not look good.
Can you move it to a function and add some code to the function that actually copies data and then call the function in the test?

Reply:
Same for the other test.

Reply:
I'm not sure if it completely addressed the concern here, but I've moved the code to a function , and called it in the test.
Pushed the same in the latest commit